src/main/java/com/targomo/client/api/request/config/multigraph/AggregationConfig.java

```diff
@@ -0,0 +1,242 @@
+package com.targomo.client.api.request.config.multigraph;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.targomo.client.Constants;
+import com.targomo.client.api.enums.MultiGraphAggregationType;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AggregationConfig {
+    private MultiGraphAggregationType type;
+    private Boolean ignoreOutlier;
```

Comment:
since you made "ignoreOutlier" and "minSourcesCount" boxed instead of native types you will have to check everywhere if their values might also be "null" . otherwise we might get NPEs later

Reply:
wanted to be consistent with what we have on TravelOptions (here: https://github.com/targomo/targomo-java/blob/master/src/main/java/com/targomo/client/api/TravelOptions.java#L121). 

We check for null values though, and we need to know sometimes if the request contains these values to be able to apply default ones if it doesn't (we do this for minSourcesCount)